VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,53 @@
+# Video 6: Working in community
+
+- What and why working within a community
+- AsyncAPI and its community
+- How to join the AsyncAPI community
+
+Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. Today we’ll talk about the AsyncAPI community.
+
+## What and why working within a community
+
+Every social group needs some common ground to meet and develop its goals. Working, as well as playing music or enjoying free time, is primarily about sharing things, thoughts, challenges, obstacles, joy, sorrow... And, after all that, find a place of understanding and, hopefully, entertainment. 
+
+A common language, customs, culture, taste, etc., are the foundations where communication and understanding can be found. Paradoxically, this is where we can find all the limits and, at the same time, all the possibilities. 
+
+However, a community is much more than a mere group of human beings linked by common characteristics or interests. A community is united by goals projected for the future, hoping to build something valuable. The projection of a reachable better place to be.
+
+But, nowadays, it's not necessary to share the same time and space to create a group of people committed to working on a project. Today the number of virtual communities is growing exponentially, letting people from all over the world exchange knowledge and ideas.
+
+We can define three main characteristics of virtual communities:
+
+•	They allow communication between users, either synchronously or asynchronously.
+•	The information can be exchanged in different formats, including images, videos, documents, audio...
+•	They promote meaningful, collaborative, and team-based learning.
+
+Beyond generating a sense of belonging, communities facilitate the exchange of information, knowledge, and experiences, contributing to their members' personal and professional development. Virtual communities are, nowadays, increasingly essential for professional and, as well, social.
+
+Human beings use somehow virtual communities to tackle and reverse the problems of physical communities virtual ones are very valuable not when shortening geographical and cultural distances, but also to restore the lack of social integration of groups discriminated within real communities.
+
+## AsyncAPI and its community
+
+As mentioned in previous videos, AsyncAPI is a specification managed and driven by its community. It’s a virtual community open to everyone who participates in the specification. Everyone is welcome. AsyncAPI is devoted to forming a community with the most accurate possible reflection of contemporary society, diverse and inclusive.
+
+Being a virtual community comprises many people from a different races, ethnicities, and cultures. As the community grows and more and more people get in, it’s increasingly broad and diverse.
```

Comment:
you mentioned `virtual` few times, but it is a first time I hear people talk about community as virtual community. Not sure if it makes sense to include this phrase